frontend/src/scenes/surveys/SurveyEdit.tsx

```diff
@@ -775,6 +779,43 @@ export default function SurveyEdit(): JSX.Element {
                                                     </>
                                                 </LemonField.Pure>
                                             )}
+                                            {featureFlags[FEATURE_FLAGS.SURVEYS_ACTIONS] && surveysActionsAvailable && (
+                                                <LemonField.Pure
+                                                    label="User performs actions"
+                                                    info="Note that these actions are only observed, and activate this survey, in the current user session."
+                                                >
+                                                    <EventSelect
+                                                        filterGroupTypes={[TaxonomicFilterGroupType.Actions]}
+                                                        itemProperty="name"
+                                                        onChange={(includedActions) => {
+                                                            setSurveyValue('conditions', {
+                                                                ...survey.conditions,
+                                                                actions: {
+                                                                    values: includedActions.map((e) => {
+                                                                        return { name: e }
```

Comment:
What's the reasoning behind using the name instead of the ID? Feels super error prone and very against how we do every other action related endpoint.

Reply:
good callout, fixed in https://github.com/PostHog/posthog/pull/23613/commits/86d515fd9b7e3ea9f2a02bfa42b0a5969eb0664a